bcda/api.go

```diff
@@ -3,13 +3,11 @@
 
  The Beneficiary Claims Data API (BCDA) allows downloading of claims data in accordance with the FHIR Bulk Data Export specification.
 
- Terms Of Service:
+ If you have a token you can use this page to explore the API.  To do this click the green "Authorize" button below and enter "Bearer {YOUR_TOKEN}"
```

Comment:
Wasn't expecting this file to change as part of BCDA-520.  Can you re-sync with `master`?

Reply:
I will.  I am just closing my other pr's right now.